spring-rabbit/src/test/java/org/springframework/amqp/rabbit/config/RabbitListenerContainerTestFactory.java

```diff
@@ -63,4 +66,13 @@ public MessageListenerTestContainer createListenerContainer(RabbitListenerEndpoi
 		return container;
 	}
 
+	@Override
+	public void setBeanName(String name) {
+		this.beanName = name;
+	}
+
+	public String getBeanName() {
+		return this.beanName;
```

Comment:
If those methods are `default` in the interface, we don't need changes to this test entity and, therefore, to the whole class.

Reply:
I think we should test method `getBeanName` in `AbstractRabbitAnnotationDrivenTests`

Reply:
I saw that there.
Thanks